mesh-workshop.bat

```diff
@@ -0,0 +1,26 @@
+@echo off
```

Comment:
This file will need to be copied over. See https://github.com/benhylau/mesh-workshop/blob/master/build#L31